File: src/app/shared/model/exemplar.ts
import { Livro } from './livro';

export class Exemplar {

    id: number; 
    numero: number; 
    disponivel: boolean; 
    livro: Livro;

    constructor(id: number, numero: number, disponivel: boolean,livro: Livro) {
        this.id=id;
        this.numero=numero;
        this.disponivel=disponivel;
        this.livro=livro;
    }
    /** 
    public getId(): number{
        return this.id;
    }
    public getNumero(): number{
        return this.numero;
    }
    public getDisponivel(): boolean{
        return this.disponivel;
    }

    public setId(id: number){
        this.id=id;
    }

    public setNumero(numero: number){
        this.numero=numero;
    }

    public setDisponivel(disponivel: boolean){
        this.disponivel=disponivel;
    } */

}

File: demo/src/main/java/com/livro/demo/model/Exemplar.java
package com.livro.demo.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "tb_exemplar")
public class Exemplar {

   @Id
   @GeneratedValue(strategy = GenerationType.AUTO)
   private Integer id;
   private Integer numero;
   private Boolean disponivel;

   public Integer getId() {
       return id;
   }

   public Integer getNumero() {
       return numero;
   }

   public Boolean getDisponivel(){
        return disponivel;
   }
 
   public void setNumero(Integer numero) {
       this.numero = numero;
   }

   public void setDisponivel(Boolean disponivel) {
       this.disponivel = disponivel;
   }
}
